### DISCUSSION TITLE
Wordpress template

### DISCUSSION THREAD

**Discussion by ybinhatem** [TDRA-ae/aegov-dls#62] [Q&A] [answered]
Greetings
I want to enquire about the release of WordPress template

**Comment by moiz-conceptlz** [comment, marked as answer]
hello @ybinhatem 

We will not be creating a complete UI template - but rather a template-based framework. This is on the roadmap and will be released soon - however, there is no definitive timeline yet.

**What does this mean?**

The underlying application architecture will use https://roots.io/bedrock/ and create a build system that will allow anyone to use a pre-created template as a parent theme. You may then, create a child theme with your own logic and UI blocks.

Elements to control the Header, Footer, Navigation, the default WordPress Search, Language management will be embedded in the primary parent theme.

**What will not be included**

As an example; the WordPress framework will not include functionality for data input or page layouts. It will still be left to the implementing entity to decide how they want to use the framework and for what purpose.

**What will be definitely included**

As a primary requirement, the WordPress template-framework will be bundled with a package.json file at a theme level and a webpack based build system. It will be configured to include classes detected within any PHP file within the theme. You will be able add the path to your child theme into the build system as well.

This will ensure that the output CSS file generated will be for the project you are implementing, pre configured to include the AE-GOV design system output.

**Comment by Ajre-it** [comment]
is there any update ?

**Reply by moiz-conceptlz** [reply]
No update as yet, however the aim is Feb 2025 end
